src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -78,7 +81,7 @@ export const KarmaScore = ({score}: KarmaScoreProps) => {
                   paddingTop: normalize(8),
                   fontSize: normalize(14),
                   lineHeight: normalize(20),
-                  color: '#4782D7',
+                  color: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `color` property of the style object has been updated. This change seems to be part of an effort to standardize the look and feel of the app. Please ensure that this change does not conflict with other parts of the application and that it adheres to the design guidelines.

```diff
-                  color: '#4782D7'
+                  color: COLORS.signed_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->